src/main/java/com/volmit/iris/core/decrees/DecIris.java

```diff
@@ -46,9 +50,9 @@
 
     private DecObject object;
 
-    @Decree(description = "Create a new world", aliases = "+")
+    @Decree(description = "Create a new world", aliases = {"+", "c"})
```

Comment:
Remove "c", its not needed

Reply:
It is, otherwise it defaults to obje**c**ts. No joke.